packages/http-client/src/client.ts

```diff
@@ -55,30 +57,63 @@ export const requestTokenRequiredClaims = ['aud', 'iss', 'exp', 'iat', 'jti']
  * @beta
  */
 export class TbdexHttpClient {
+
   /**
-   * sends the message provided to the intended recipient
-   * @param opts - options
+   * Sends an RFQ and options to the PFI to initiate an exchange
+   * @param rfq The RFQ message that will be sent to the PFI
+   * @param opts.replyTo A callback URL where the PFI will send subsequent messages
+   * @throws if message verification fails
+   * @throws if recipient DID resolution fails
+   * @throws if recipient DID does not have a PFI service entry
+   */
+  static async createExchange(rfq: Rfq, opts?: { replyTo?: string }): Promise<void> {
+    await rfq.verify()
+
+    const { to: pfiDid, exchangeId } = rfq.metadata
+    const requestBody = JSON.stringify({ rfq, replyTo: opts?.replyTo })
+
+    await TbdexHttpClient._sendMessage(pfiDid, `/exchanges/${exchangeId}/rfq`, requestBody)
+  }
+
+  /**
+   * Sends the Order message to the PFI
+   * @param order The Order message that will be sent to the PFI
+   * @throws if message verification fails
+   * @throws if recipient DID resolution fails
+   * @throws if recipient DID does not have a PFI service entry
+   */
+  static async submitOrder(order: Order): Promise<void> {
+    await order.verify()
+
+    const { to: pfiDid, exchangeId } = order.metadata
+    const requestBody = JSON.stringify(order)
+
+    await TbdexHttpClient._sendMessage(pfiDid, `/exchanges/${exchangeId}/order`, requestBody)
+  }
+
+  /**
+   * Sends the Close message to the PFI
+   * @param close The Close message that will be sent to the PFI
    * @throws if message verification fails
    * @throws if recipient DID resolution fails
    * @throws if recipient DID does not have a PFI service entry
    */
-  static async sendMessage<T extends Message>(opts: SendMessageOptions<T>): Promise<void> {
-    const { message, replyTo } = opts
+  static async submitClose(close: Close): Promise<void> {
+    await close.verify()
+
+    const { to: pfiDid, exchangeId } = close.metadata
 
-    await message.verify()
+    const requestBody = JSON.stringify(close)
 
-    const { to: pfiDid, exchangeId, kind } = message.metadata
+    await TbdexHttpClient._sendMessage(pfiDid, `/exchanges/${exchangeId}/close`, requestBody)
+  }
+
+  private static async _sendMessage(pfiDid: string, path: string, requestBody: string): Promise<void> {
```

Comment:
Just an idea, if we actually want to make this private at runtime (as opposed to build time) then 

```suggestion
  static async #sendMessage(pfiDid: string, path: string, requestBody: string): Promise<void> {
```

Not sure it matters much either way

Reply:
Since we're using `private` rather than `#` syntax throughout the SDK, I'd prefer to leave it for now. But if we want to update it throughout the SDK, I'm open to doing that in another PR

Reply:
I decided to remove the underscore from the method name though.